### ISSUE TITLE
Add Unit Tests for components

### ISSUE THREAD

**Issue by Sachin-chaurasiya** [ashutosh1919/react-awesome-shapes#84]
Currently, we have 0% test coverage, we should add some unit tests for components so that we can easily debug if something is breaking in components.
cc: @ashutosh1919

**Comment by ashutosh1919** [comment]
@Sachin-chaurasiya yes i definitely agree with you. I have little experience with setting up testing with `jest` in react. @Sachin-chaurasiya, Can you setup basic test? Then we all can start contributing over it.

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya yes i definitely agree with you. I have little experience with setting up testing with `jest` in react. @Sachin-chaurasiya, Can you setup basic test? Then we all can start contributing over it.

@ashutosh1919 , Yeah sure.

**Comment by ShaileshParmar11** [comment]
Hi, @ashutosh1919 & @Sachin-chaurasiya I would like to take participate in adding unit tests for the project!

**Comment by Sachin-chaurasiya** [comment]
Go ahead @ShaileshParmar11 .
